code/modules/hydroponics/fermenting_barrel.dm

```diff
@@ -1,78 +1,54 @@
 /obj/structure/fermenting_barrel
-	name = "wooden barrel"
-	desc = "A large wooden barrel. You can ferment fruits and such inside it, or just use it to hold liquid."
-	icon = 'icons/obj/objects.dmi'
-	icon_state = "barrel"
+	name = "brewing barrel"
```

Comment:
If you think it's a good idea, you could make a separate craft-able barrel that *just* has the reagent storage mechanisms applied to retain the craft-able "large barrel" format. It'd also keep the fancy sprite that was made for me in-game, which I adore.

Reply:
i don't really see the utility in this when one is available via plumbing already

the barrel sprite is nice tho, i'll see about finding a use for it